packages/dd-trace/test/appsec/reporter.spec.js

```diff
@@ -201,24 +210,26 @@ describe('reporter', () => {
     })
 
     it('should not overwrite origin tag', () => {
-      const req = stubReq({ '_dd.origin': 'tracer' })
+      span.context()._tags = { '_dd.origin': 'tracer' }
```

Comment:
Interesting. Object mutation is like Pi, it has many uses where you least expect it.